### PR TITLE
logger: Add logging profile to support comparison of multiple sensors

### PR DESCRIPTION
**Pull request by priseborough**
This is required to assist users with diagnosing pre-arm sensor consistency failures.

### CODE REVIEW

File: src/modules/logger/logger.cpp [hunk 1]
```diff
@@ -624,6 +624,14 @@ void Logger::add_thermal_calibration_topics()
 	add_topic("sensor_gyro", 100);
 }
 
+void Logger::add_sensor_comparison_topics()
+{
+	add_topic("sensor_accel", 10);
```
Comment (bkueng) [inline src/modules/logger/logger.cpp]: Just to clarify: 10 is the interval in ms, not Hz. Is that your intention?

Comment (priseborough) [reply src/modules/logger/logger.cpp]: No it wasn't, i will change it to 100.

**Comment by bkueng** [comment]
@priseborough CI failed because of flash overflow. Can you please rebase to master?

**Comment by priseborough** [comment]
OK

**Pull request merged**